settings.php

```diff
@@ -72,6 +72,12 @@ function page_optimize_settings_field_css( $args ) {
 			<input type="checkbox" id="page_optimize-css" name="page_optimize-css" value="1" <?php checked( get_option( 'page_optimize-css' ) ); ?>>
 			<?php _e( 'Concatenate styles', page_optimize_get_text_domain() ); ?>
 		</label>
+		<br />
```

Comment:
In regular HTML, we don't need to self-close `<br>` tags. No need to change. Just a comment.
https://html.spec.whatwg.org/#the-br-element

We should also be using CSS instead of `<br>` to separate elements on different lines, but that's a small thing that can be fixed later. I think I may have been the first to use `<br>` here btw... just sharing my thoughts.

Reply:
Very true. :)